apps/docs/content/guides/auth/phone-login/textlocal.mdx

```diff
@@ -0,0 +1,69 @@
+---
+id: 'auth-textlocal'
+title: 'Phone Auth with Textlocal'
+description: 'How to set up and use Mobile OTP with Textlocal and Supabase.'
+---
+
+<Admonition type="tip">
+
+To manage costs related to SMS authentication, consider setting project rate limits and configuring CAPTCHA. For additional cost-control strategies and to ensure compliance with TRAI's DLT requirements, visit the Production Checklist.
+
+</Admonition>
+
+## Prerequisites
+
+Before you begin, ensure you have the following:
+
+- **A Textlocal account**: Sign up [here](https://www.textlocal.com/signup) to start sending SMS.
+- **A Supabase project**: Necessary for SMS authentication integration. Create your project [here](https://supabase.com/dashboard).
+- **A mobile phone**: To receive SMS, verifying the setup and functionality.
+- **DLT Registration**: Required for businesses in India, as per TRAI regulations, to send commercial SMS.
```

Comment:
Can you take the DLT registration instructions out in favor of the admonition above? ☝🏼 

We don't have the capacity to keep up-to-date instructions on how to comply with government regulations, and don't want to provide potentially outdated ones, especially given the number of countries we serve.